test/spec/directives/numbers-only.spec.js

```diff
@@ -68,5 +68,37 @@ describe('numbersOnly', function() {
         expect(element.val()).toBe('53.51');
       });
     });
+
+    describe('When is a number no has decimal values', function() {
```

Comment:
`When input number is an integer` or `When input number doesn't have decimal values` would be better